### PR TITLE
Cnfcert 30 changes

### PR DESCRIPTION
**Pull request by isaacdorfman**

### CODE REVIEW

**Review by jun-rh — DISMISSED** [review]

**Review by HamadiSE — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -3,6 +3,9 @@
 
 install: 
 	bash ./deploy-all.sh
+
```
Comment (edcdavid) [inline Makefile:6]: For readability: rename the deploy-all.sh script to deploy-test-pods.sh (or similar), remove the "bash deploy-partner-pods.sh" line. Call both scripts in "install section" of Makefile. Create a new Makefile section for deploying test pods.

**Review by edcdavid — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (isaacdorfman) [reply Makefile:6]: I fixed this

**Review by jun-rh — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (HamadiSE) [reply Makefile:6]: this removes the ability to do a simple #make install to setup everything.
can we have two options ? make install all and make install partners

File: deploy-partner-pods.sh [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+if [[ -n "${TNF_MINIKUBE_ONLY}" ]] && [[ ${TNF_MINIKUBE_ONLY} = "true" ]]; then
+    echo "Running with Minikube, no FsDiff pod created"
+else
+    export res=$(oc get deployment -n $NAMESPACE_TO_GENERATE node-master 2>&1)
+    if [[ ${res#Error from server (NotFound)} != ${res} ]] || [[ ${res#No resources found} != ${res} ]]; then
+        cat ./local-test-infra/fsdiff-deployment.yaml | ./script/mo > ./temp/rendered-fsdiff-template.yaml
+        oc apply -f ./temp/rendered-fsdiff-template.yaml
+        rm ./temp/rendered-fsdiff-template.yaml
+    else
```
Comment (jun-rh) [inline deploy-partner-pods.sh:24]: Thinking more on this, it seems to be that skipping is probably not what we want for the container case at least. The container may contain a different versions of partner pods. Common use case would be clean the deployments/pods already there in the namespace and redeploy with the version from the container image. @isaacdorfman @edcdavid @HamadiSE @wying3 Comments?

File: local-test-infra/fsdiff-deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: node-master
+  namespace: {{ NAMESPACE_TO_GENERATE }}
```
Comment (greyerof) [inline local-test-infra/fsdiff-deployment.yaml:6]: Please add the label "test-network-function.com/generic: fs_diff_master" also to the deployment's metadata so this deployment can be easily filtered out by the TNF.

File: deploy-partner-pods.sh (hunk 2 above)
Comment (isaacdorfman) [reply deploy-partner-pods.sh]: I did this

File: local-test-infra/fsdiff-deployment.yaml (hunk 3 above)
Comment (isaacdorfman) [reply local-test-infra/fsdiff-deployment.yaml:6]: Done

**Review by jun-rh — APPROVED** [review]

**Review by HamadiSE — APPROVED** [review]

**Review by edcdavid — APPROVED** [review]

**Pull request merged**